### DISCUSSION TITLE
[v3] Applying validation rule for an array property and it's children

### DISCUSSION THREAD

**Discussion by addtobag** [livewire/livewire#5928] [1. Help]
I'm trying to apply a required rule to a property called $options and it's children, but i'm getting an error returning of:

**No property found for validation: [options.*]**

Am i missing something?

```
#[Rule([
    'options.*' => 'required',
])]
public $options = [];
```

**Comment by joshhanley** [comment]
@addtobag hmm can you try type hinting $options with `Array` and seeing if that works?

**Reply by madushancs** [reply]
Did you find any solution?

**Comment by jleonardolemos** [comment]
Same problem here :(
